### PR TITLE
Adds container types INJECTABLE and DRAWABLE

### PR DESCRIPTION
**Pull request by ghost**
You can only draw from DRAWABLE containers. You can draw and inject to INJECTABLE containers. You can draw and inject to OPENCONTAINER containers. It also checks for mobs.

### CODE REVIEW

File: code/__DEFINES/reagents.dm [hunk 1]
```diff
@@ -2,7 +2,9 @@
 #define LIQUID 2
 #define GAS 3
 
-#define OPENCONTAINER	4096	// is an open container for chemistry purposes
+#define INJECTABLE		1024	//Makes reagents addable through droppers and syringes
+#define DRAWABLE		2048	//If a syringe can draw from it
```
Comment (Cyberboss) [inline code/__DEFINES/reagents.dm]: Make DRAWABLE 1024 and INJECTABLE 2048 | DRAWABLE

File: code/game/atoms.dm [hunk 2]
```diff
@@ -192,6 +192,19 @@
...
+	if(isliving(src) && allowmobs)
+		var/mob/living/L = src
+		return L.can_inject()
+	if(container_type & OPENCONTAINER)
+		return TRUE
+	return container_type & INJECTABLE
+
+/atom/proc/is_drawable(allowmobs = TRUE)
+	if(is_injectable(allowmobs)) //Everything that can be injected can also be drawn from, but not vice versa
+		return TRUE
```
Comment (Cyberboss) [inline code/game/atoms.dm]: And then you won't need this check

**Review by Cyberboss — DISMISSED** [review]

File: code/__DEFINES/reagents.dm (hunk 1 above)
Comment (Really-Good-Soda-Flavor) [inline code/__DEFINES/reagents.dm]: Should I make OPENCONTAINER include those, too?

Comment (Cyberboss) [inline code/__DEFINES/reagents.dm]: yes

Comment (lzimann) [inline code/__DEFINES/reagents.dm]: Aren't containers using any of these flag slots or was it just a lazy change leaving OPENCONTAINER at 4096?

File: code/game/atoms.dm (hunk 2 above)
Comment (AnturK) [inline code/game/atoms.dm]: I actually don't see much purpose in conjoining these ? One way injectable doesn't seem that far fetched ?

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**